src/components/app/index.js

```diff
@@ -11,32 +11,21 @@ import Metronome from '../../utils/metronome';
 import Dialog from '../dialog';
   // temporary, will be deleted with #347
 
-const App = () => {
-  // start dispatching sync ticks
-  const metronome = new Metronome();
-  metronome.init();
+// start dispatching sync ticks
+const metronome = new Metronome();
+metronome.init();
 
-  return (
-    <section className={styles['body-wrapper']}>
-      <Header />
-      <main className=''>
-        <Route path="/main" render={({ match }) => (
-          <main className=''>
-            <Account />
```

Comment:
We don't need to render Account component and Link components every time that we change the route.They are the same in forging and voting and transaction and rendering them over and over is useless and it will decrease performance of our application .So, we must use nesting routes to solve this problem.why do you remove nested routes?